symbolic-debuginfo/src/dwarf.rs

```diff
@@ -43,6 +43,10 @@ type IncompleteLineNumberProgram<'a> = gimli::read::IncompleteLineProgram<Slice<
 type LineNumberProgramHeader<'a> = gimli::read::LineProgramHeader<Slice<'a>>;
 type LineProgramFileEntry<'a> = gimli::read::FileEntry<Slice<'a>>;
 
+fn offset(addr: u64, offset: i64) -> u64 {
+    (addr as i128).saturating_sub(offset as i128) as u64
```

Comment:
While the `saturating_sub` is strictly better, can you explain the reasoning for this change?